test/e2e/run-all.sh

```diff
@@ -64,3 +64,12 @@ concurrently --kill-others \
   'yarn dapp' \
   'sleep 5 && mocha test/e2e/address-book.spec'
 
+export GANACHE_ARGS="${BASE_GANACHE_ARGS} --deterministic --account=0x53CB0AB5226EEBF4D872113D98332C1555DC304443BEE1CF759D15798D3C55A9,25000000000000000000"
+  concurrently --kill-others \
+    --names 'ganache,dapp,e2e' \
+    --prefix '[{time}][{name}]' \
+    --success first \
+    'node test/e2e/mock-3box/server.js' \
+    'yarn ganache:start' \
+    'yarn dapp' \
+    'sleep 5 && mocha test/e2e/threebox.spec'
```

Comment:
Interesting that the end-of-file newline is missing. Our `.editorconfig` file does ask for one - maybe your editor doesn't have the editorconfig plugin installed?

Reply:
fixed in acd4f9bc7